frontend/src/components/TaskValueCreatedVisualization.tsx

```diff
@@ -14,23 +14,37 @@ import { apiV2 } from '../api/api';
 
 const classes = classNames.bind(css);
 
+// either provide versions or customer stakes
 export const TaskValueCreatedVisualization: FC<{
-  versions: Version[];
+  stakes?: CustomerStakes[];
+  versions?: Version[];
```

Comment:
```suggestion
  ({ stakes: CustomerStakes[] } | { versions: Version[] }) & {
```

The "either" could be expressed in the type, obsoleting the comment